scripts/classificationloss.py

```diff
@@ -0,0 +1,220 @@
+import sys, os
+
+from PIL import Image
+from podm import metrics
+import numpy as np
+
+# Accroding to: https://pyimagesearch.com/2022/05/02/mean-average-precision-map-using-the-coco-evaluator/
+# This ia also helpful: https://github.com/yfpeng/object_detection_metrics/tree/master/src/podm
+
+
+def IOU(boxes1, boxes2):
```

Comment:
What does this do that cannot already be done with existing tools?
https://github.com/yfpeng/object_detection_metrics/blob/3801acc32d052d1fbf1565f96a9f8d390701f911/src/podm/box.py#L91

Then we can make use of the existing COCO dataset format as well.